bindings/cpp/include/wtf/platform/platform_aux_pthreads_threaded_impl.h

```diff
@@ -39,17 +35,19 @@ void PlatformSetThreadLocalEventBuffer(EventBuffer* event_buffer) {
 }
 
 std::string PlatformGetThreadName() {
-#if defined(__APPLE__)
```

Comment:
why remove this? the nice thing about the threadid here is that it matches other apple tooling and system traces, where as pthread_self does not. I'm not tied to this as I haven't done anything where I needed to match them up in awhile, but at a time it was useful to be able to correlate events between them, so FYI.